src/server/db_slice.h

```diff
@@ -409,6 +413,13 @@ class DbSlice {
   void CreateDb(DbIndex index);
   size_t EvictObjects(size_t memory_to_free, PrimeIterator it, DbTable* table);
 
+  enum class FindInternalMode {
+    kUpdateStats,
```

Comment:
nit: as this is now only relevant to hit/miss stats maybe call it UpdateCacheStats

Reply:
Lovely!